### ISSUE TITLE
Problems with fetching entries by content type (bug?)

### ISSUE THREAD

**Issue by a-lehdet** [contentful/contentful-management.rb#160]
Hi,

I've noticed some weird behaviour when attempting to fetch entries by content type like this:

```
require "contentful/management"

client = Contentful::Management::Client.new("<TOKEN>")
environment = client.environments("<SPACE-ID>").find("master")

environment.entries.all(content_type: "theme", limit: 10).each do |theme|
  puts theme.id
end
```

I can determine from the entry IDs printed out, that the entries in question are of *ALL* content types, not of "theme" as requested. Furthermore, the limit is fixed to 100, no matter what I set as the `limit:` parameter value.

Before environments where introduced, the comparable functionality would work just fine like this:

`space.entries.all(content_type: "theme", limit: 10)`

I can get it to work like this, but I didn't figure out how to set a limit.

```
environment.content_types.find("theme").entries.all.each do |theme|
  puts theme.id
end
```

Am I doing something wrong or is this a genuine bug?

**Comment by dlitvakb** [comment]
Hey @a-lehdet,

Let me look into this and come back to you,

Cheers

**Comment by dlitvakb** [comment]
Hey @a-lehdet,

This indeed is a bug.

Looking into it,

Cheers

**Comment by dlitvakb** [comment]
Hey @a-lehdet,

This has just been fixed on release 2.1.1

Cheers

**Comment by a-lehdet** [comment]
Excellent @dlitvakb, thanks!
